plugins/formal-verification/testData/diagnostics/verifies/inlining/custom_run_functions.kt

```diff
@@ -51,7 +51,7 @@ public fun <!VIPER_TEXT!>useRun<!>() {
     val doubleIntRunResult = 1.doubleIntRun { plus(1) } == 3
     val genericReceiverResult = 1.copiedRun { plus(2) } == 3
 
-    verify(
+    <!PURITY_VIOLATION!>verify(
```

Comment:
Please introduce helper variables.

Reply:
Sorry, to clarify: please introduce helper variables so that the tests pass, rather than adding purity violations.